File: userManager/python/account.py
class account:
    # constructor, default,
    def __init__(self, user_name = "", password = ""):
        self.user_name = user_name
        self.password = password

    # method, mutator, 
    def set_user_name(self, user_name):
        self.user_name = user_name
    def set_password(self, password):
        self.password = password

    # method, accessor, 
    def get_user_name(self):
        return self.user_name
    def get_password(self):
        return self.password
    
    # method, accessor, print account information
    def printer(self, string, method):
        return print(string, method)
pass

File: userManager/java/account.java
public class account 
{
    // variables, declare, initialise,
    private String email = "";
    private String password = "";
    private String firstName = "";
    private String lastName = "";
    private static int numberOfAccounts = 0;

    // construct, object, null,
    public account()
    {
        setAccountCounter();
    }

    // construct, object, parameters,
    public account(String email, String password)
    {
        this.email = email;
        this.password = password;
        setAccountCounter();
    }

    // access, object, parameters,
    public String getEmail()
    {
        return this.email;
    }
    public String getPassword()
    {
        return this.password;
    }
    public int getAccountCounter()
    {
        return numberOfAccounts;
    }

    // update, class, parameter,
    private static void setAccountCounter()
    {
        numberOfAccounts++;
    }
    // update, object, parameters,
    public void setFirstName(String firstName)
    {
        this.firstName = firstName;
    }
    public void setLastName(String lastName)
    {
        this.lastName = lastName;
    }
    public void setEmail(String username)
    {
        this.email = username;
    }
    public void setPassword(String password)
    {
        this.password = password;
    }
    // update, object, printer,
    public void setPrinter() 
    {
        System.out.println(getEmail());    
        System.out.println(getPassword());
        System.out.println(getAccountCounter());
    }
}
